tests/features/subscriptions.feature

```diff
@@ -0,0 +1,58 @@
+@api
+Feature: Subscription
+  In order to manage my subscriptions on the website
+  As an authenticated user
+  I want to be able to subscribe to content and manage subscriptions.
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+    And the module is enabled
+      |modules                      |
+      |multisite_notifications_core |
+  @javascript
+  Scenario: Create a page and have someone register to it
+    And I go to "admin/config/system/site-information_en"
+    Then I fill in "E-mail address" with "automated-notifications@nomail.ec.europa.eu"
+    And I select "01000" from "classification"
+    And I press "Save configuration"
+    Then I go to "node/add/page"
```

Comment:
"Then" should be used to observe outcomes. Use "When" instead to describe transitions the user performs.